pkg/master/import_known_versions_test.go

```diff
@@ -67,6 +67,63 @@ func TestTypeTags(t *testing.T) {
 	}
 }
 
+func TestRegisteredObjectFieldSelector(t *testing.T) {
+	namespaceScopedObjects := map[string][]string{
```

Comment:
@liggitt, while using api.Scheme.AllKnownTypes(), I can see there are lot of non-API exposed objects like `v1.GetOptions`, `v1.List`, in my mind, for these kinds of objects, I don't think we need to set default field selector conversion function for them, right?